### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ArsenSkandalenko**
[DEMO LINK](https://ArsenSkandalenko.github.io/react_todo-app-add-and-delete/)

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${filterBy === Status.Completed ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilterBy(Status.Completed)}
+        >
+          Completed
+        </a>
```
Comment (GUSILLUS) [inline src/components/Footer/Footer.tsx:53]: Consider using `Object.values(Status)` and `map()` method

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+      <form onSubmit={handleAddTodo}>
+        <input
+          ref={titleField}
+          data-cy="NewTodoField"
+          type="text"
+          className="todoapp__new-todo"
+          placeholder="What needs to be done?"
+          value={title}
+          onChange={event => setTitle(event.target.value.trimStart())}
```
Comment (GUSILLUS) [inline src/components/Header/Header.tsx:38]: Consider creating handler function here

File: src/components/TodoItem/TodoItem.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  handleDeleteTodo,
+  isLoading,
+}) => {
+  return (
+    <div
+      key={todo.id}
```
Comment (GUSILLUS) [inline src/components/TodoItem/TodoItem.tsx:18]: Consider using destruction assignment for todo object

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ!

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (vadiimvooo) [reply src/components/Footer/Footer.tsx:53]: Still not fixed.

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job, but some of previous coments still not fixed. Please feel free to ask for help in fe_chat.

**Pull request closed without merging**